src/renderer/components/FormattedDate.js

```diff
@@ -7,6 +7,7 @@ type Props = {
   format?: string,
 };
 
+/* TODO: Replace this function with the useDateTimeFormat hook */
```

Comment:
the hook exists in the PR, is this still necessary ?

Reply:
The plan is to replace this function (FormattedDate) with the new hook introduced by this PR to move from the `moment` library to the native `Intl` Javascript API.

Reply:
great